src/components/NavBar/AccountMenu.tsx

```diff
@@ -1,18 +1,35 @@
 import { useAuth0 } from "@auth0/auth0-react";
-import { Avatar, Link } from "@mui/material";
+import { Avatar, Box, Link } from "@mui/material";
 import { Typography } from "@mui/material";
 import { Button } from "@mui/material";
 import Popover from "@mui/material/Popover";
 import Stack from "@mui/material/Stack/Stack";
 import React from "react";
 import { useAppSelector } from "../../state/hooks";
+import LogoutIcon from "@mui/icons-material/Logout";
+import ArrowDropDownIcon from "@mui/icons-material/ArrowDropDown";
 
 const styles = {
   button: {
     marginTop: "auto",
     marginBottom: 2,
   },
   nameContainer: { paddingLeft: 2, textAlign: "left" },
+  popover: {
+    paddingTop: "10px",
+    paddingBottom: "10px",
+    borderRadius: "10px",
```

Comment:
Can we please make these numbers so they match with our theme spacing variables? Basically, any time you specify a raw number in the `sx` prop, that number gets multiplied by the theme spacing variable ((in our case 8 pixels) and that's the amount of pixels that gets set for that property.

For example, if you do `padding: 1`, that means a padding of 1 * 8 pixels = 8 pixels. If you do `padding: 2`, that means a padding of 2 * 8 pixels = 16 pixels. This is done so that we have consistent spacing units across the app. I'd suggest something like this:
```suggestion
    padding: 1,
    borderRadius: 3,
```

Reply:
Updated as per your suggestions mentioned above